app_legacy/Site/AppServiceProvider.php

```diff
@@ -16,9 +20,19 @@ public function boot(): void
     {
         if ($this->app->runningInConsole()) {
             $this->commands([
+                LogUsersOnlineCount::class,
+                DeleteExpiredEmailVerificationTokens::class,
+                DeleteOverdueUserAccounts::class,
             ]);
         }
 
+        $this->app->booted(function () {
+            $schedule = $this->app->make(Schedule::class);
+            $schedule->command('ra-legacy:site:log-online-users-count')->everyThirtyMinutes();
+            $schedule->command('ra-legacy:site:delete-expired-email-verification-tokens')->daily();
+            $schedule->command('ra-legacy:site:delete-overdue-user-accounts')->daily();
```

Comment:
Why use strings here and ::class in Platform/AppServiceProvider?

Reply:
Good catch, thanks! Missed that, corrected.